### PR TITLE
tests: add api check to warn if there are refs in status

### PR DESCRIPTION
**Pull request by justinsb**
We only want refs in spec; in the status we expect "full links" in the
"external" format.

### CODE REVIEW

File: tests/apichecks/crds_test.go [hunk 1]
```diff
@@ -80,6 +80,53 @@ func TestMissingRefs(t *testing.T) {
...
+				if fieldPath == ".status.externalRef" {
+					return
+				}
+
+				// Check if this is named like a ref
+				isRef := false
+				if strings.HasSuffix(fieldPath, "Ref") {
+					isRef = true
+				}
+				if strings.HasSuffix(fieldPath, "Refs[]") || strings.HasSuffix(fieldPath, "Refs") {
```
Comment (yuwenma) [inline tests/apichecks/crds_test.go:112]: +1. This is why we want <kind>Refs!

**Review by yuwenma — COMMENTED** [review]
/lgtm
/approve

Thanks!

**Comment by yuwenma** [comment]
bump this PR. (I just realized it is not merged) 
@justinsb Do you mind rebasing this PR. I think the unit-test failure has been fixed.

**Review by justinsb — COMMENTED** [review]
Rebased (after I fixed the incorrect ref) so should now pass!+

**Comment by yuwenma** [comment]
/lgtm

Thank you so much!

**Pull request merged**